recipes-extended/realsense-camera/realsense-camera_1.8.0.bb

```diff
@@ -2,14 +2,13 @@ DESCRIPTION = "ROS driver for RealSense camera"
 LICENSE = "BSD"
 LIC_FILES_CHKSUM = "file://package.xml;beginline=9;endline=9;md5=4bcd5f0e140091c68f8c0bcfdd6181e0"
 
-DEPENDS = "librealsense cv-bridge camera-info-manager pcl-ros roscpp"
+DEPENDS = "librealsense cv-bridge camera-info-manager pcl-ros roscpp rgbd-launch"
```

Comment:
Probably you wanted to add `rgdb-launch` to the `RDEPENDS` field, not `DEPENDS`?